content/docs/command-reference/dag.md

```diff
@@ -81,28 +64,10 @@ export DVC_PAGER=more
 
 ## Examples
 
-Default mode: show stage files that `output.dvc` recursively depends on:
-
-```dvc
-$ dvc pipeline show output.dvc
-raw.dvc
-data.dvc
-output.dvc
-```
-
-The same as previous, but show commands instead of DVC-files:
-
-```dvc
-$ dvc pipeline show output.dvc --commands
-download.py s3://mybucket/myrawdata raw
-cleanup.py raw data
-process.py data output
-```
-
 Visualize DVC pipeline:
 
 ```dvc
-$ dvc pipeline show eval.txt.dvc --ascii
+$ dvc dag eval.txt.dvc
```

Comment:
@efiop this command and output do not look correct to me for DVC 1.0

Reply:
Styling of the boxes is the only difference, and it was different long before 1.0. Structurally it is still correct. I would delete this for now, not to waste time on re-generating this and would take from get-started. But could just put whatever too as a placeholder for now. WIll take a look.

Reply:
it's not about styling only as fr as I udnerstand

`eval.txt.dvc` should be removed?

stage names should not include `.dvc` anymore I guess

Reply:
@shcheklein Took a new example instead. Fixed. Thanks!